e2e/cypress/integration/crash_spec.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/// <reference path="../support/index.d.ts" />
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import '@testing-library/cypress/add-commands';
```

Comment:
Please move this to `cypress/support/index.ts` so it don't need to import in each spec file. Same for other instance.

Reply:
done. [`319c6be` (#130)](https://github.com/mattermost/mattermost-plugin-demo/pull/130/commits/319c6be763d6a8d02bb2da9abdd9c9e51b170926)